### PR TITLE
Add vagga integration (sqlite version)

### PR DESCRIPTION
**Pull request by klochenok**
Integrate with vagga #5

### CODE REVIEW

File: vagga.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+minimum-vagga: v0.7.0
+containers:
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.9, <2.0'
+      - 'gunicorn >=19.6.0, <20.0'
```
Comment (DrEdi) [inline vagga.yaml]: You can write just 19.6

File: vagga.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - 'whitenoise >=3.2.2, <4.0'
+      - 'django-environ >=0.4.1, <1.0'
```
Comment (DrEdi) [inline vagga.yaml]: Here also better to leave just to the minor version.
Read very good article about this [here](http://semver.org/#semantic-versioning-200)

**Review by DrEdi — CHANGES_REQUESTED** [review]
Fix versions of deps, please.

File: vagga.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    - !Sh pip freeze > requirements.txt
+  django:
```
Comment (webknjaz) [inline vagga.yaml]: It would be nice to have `environ-file` in this container declaration

File: vagga.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  pip-freeze:
+    setup:
+    - !Alpine v3.4
+    - !Py3Install
+      - pip
+      - 'Django >=1.9, <2.0'
+      - 'gunicorn >=19.6.0, <20.0'
+      - 'whitenoise >=3.2.2, <4.0'
+      - 'django-environ >=0.4.1, <1.0'
+    - !Sh pip freeze > requirements.txt
```
Comment (webknjaz) [inline vagga.yaml]: Keep this file the same as it is now (including other files)

File: vagga.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  django:
+    setup:
+    - !Alpine v3.4
+    - !PipConfig
+      dependencies: true
+    - !Py3Requirements requirements.txt
+commands:
+  run: !command
```
Comment (webknjaz) [inline vagga.yaml]: Have you tested this?

File: vagga.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    description: Start server
+    container: django
+    run: python3 manage.py runserver
+  manage.py:
```
Comment (webknjaz) [inline vagga.yaml]: and this?

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: vagga.yaml (hunk 4 above)
Comment (klochenok) [reply vagga.yaml]: Do you mean to divide requirements file like i did it before?

Comment (webknjaz) [reply vagga.yaml]: yes

File: .gitignore [hunk 7]
```diff
@@ -6,3 +6,4 @@ db.sqlite3
 .DS_Store
 .python-version
 .env
+.vagga
```
Comment (MaxymVlasov) [inline .gitignore]: I need newline at end of file.
Give me it)

**Review by DrEdi — APPROVED** [review]

File: vagga.yaml [hunk 8]
```diff
@@ -3,25 +3,36 @@ containers:
...
     setup:
     - !Alpine v3.4
+    - &django-build !BuildDeps
+      - postgresql-dev
     - !Py3Install
       - pip
       - 'Django >=1.9, <2.0'
-      - 'gunicorn >=19.6.0, <20.0'
+    - !Sh pip freeze > requirements/common.txt
+    - !Sh pip uninstall -r requirements/common.txt
```
Comment (webknjaz) [inline vagga.yaml]: I guess you are missing `--yes` here

**Review by webknjaz — CHANGES_REQUESTED** [review]
Check yourself first before submitting for review

**Review by webknjaz — APPROVED** [review]

**Review by pavlenk0 — APPROVED** [review]

**Pull request merged**